### ISSUE TITLE
Keyboard focus order is not proper on "Description" and "Create github pull request" screen: A11y_Visual Studio Code Web Extensions_Github Pull request_Description_Keyboard

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode-pull-request-github#6451]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Visual Studio Code Web Extensions; #BM_Visual Studio Code Web Extensions_Web_Oct2024; #DesktopWeb; #A11yMAS; #WCAG2.4.3; #FocusOrder; #Win11; #FTP; #Chrome; #A11yAuto;

### Environment and OS details:
Application name: Visual Studio Code Web Extensions
Chrome Version: Version 130.0.6723.92 (Official Build) (64-bit)
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github

### Reproduction Steps:

1. Open url https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
2. TAB to 'Github Pull request' and press enter.
3. TAB to "All open" control and hit ENTER.
4. TAB to "Description" control and hit ENTER.
5. Observe the issue whether Keyboard focus order is proper on "Description" and "Create github pull request" screen or not.

### Actual behavior:
Keyboard focus order is not proper on "Description" and "Create github pull request" screen.

### Expected behavior:
Keyboard focus order should be proper i.e. from left to right and top to bottom order on "Description" and "Create github pull request" screen.

### User Impact:
Keyboard only users will not be able to understand correct flow of the page.

### Attachment:
![Image](https://github.com/user-attachments/assets/854bf56e-7428-442a-9461-b3271e1b6525)
![Image](https://github.com/user-attachments/assets/6c5f5603-62cf-4462-a2e0-2e52cbeae0d6)
